packages/daemon/src/daemon.js

```diff
@@ -420,84 +488,23 @@ const makeEndoBootstrap = async (
-      const external = makePetStoreInspector(storeFormulaIdentifier);
-      return { external, internal: undefined };
-    } else if (formulaType === 'pet-store') {
-      const external = petStorePowers.makeIdentifiedPetStore(
-        formulaNumber,
-        assertPetName,
-      );
-      return { external, internal: undefined };
-    } else if (formulaType === 'host') {
-      const workerFormulaNumber = derive(formulaNumber, 'worker');
-      const workerFormulaIdentifier = `worker:${workerFormulaNumber}`;
-      const inspectorFormulaNumber = derive(formulaNumber, 'pet-inspector');
-      const inspectorFormulaIdentifier = `pet-inspector:${inspectorFormulaNumber}`;
-      // Note the pet store formula number derivation path:
-      // root -> host -> inspector -> pet store
-      const storeFormulaNumber = derive(inspectorFormulaNumber, 'pet-store');
-      const storeFormulaIdentifier = `pet-store:${storeFormulaNumber}`;
-
-      // Behold, recursion:
-      // eslint-disable-next-line no-use-before-define
-      return makeIdentifiedHost(
-        formulaIdentifier,
-        endoFormulaIdentifier,
-        storeFormulaIdentifier,
-        inspectorFormulaIdentifier,
-        workerFormulaIdentifier,
-        leastAuthorityFormulaIdentifier,
-        context,
-      );
-    } else if (formulaType === 'endo') {
-      if (formulaNumber !== rootNonce) {
-        throw new Error('Invalid endo formula number.');
-      }
-
-      // TODO reframe "cancelled" as termination of the "endo" object and
-      // ensure that all values ultimately depend on "endo".
-      // Behold, self-referentiality:
-      // eslint-disable-next-line no-use-before-define
-      return { external: endoBootstrap, internal: undefined };
-    } else if (formulaType === 'least-authority') {
-      return { external: leastAuthority, internal: undefined };
-    } else if (formulaType === 'web-page-js') {
-      if (persistencePowers.getWebPageBundlerFormula === undefined) {
-        throw Error('No web-page-js formula provided.');
-      }
-      // Note that this worker is hardcoded to be the "MAIN" worker of the
-      // default host.
-      const workerFormulaNumber = derive(defaultHostFormulaNumber, 'worker');
-      const workerFormulaIdentifier = `worker:${workerFormulaNumber}`;
-
-      return makeControllerForFormula(
-        'web-page-js',
-        derive(formulaNumber, 'web-page-js'),
-        persistencePowers.getWebPageBundlerFormula(
-          workerFormulaIdentifier,
-          defaultHostFormulaIdentifier,
-        ),
-        context,
-      );
-    } else if (
+    if (
       [
+        'endo',
+        'worker',
         'eval',
+        'readable-blob',
         'make-unconfined',
         'make-bundle',
+        'host',
         'guest',
+        'least-authority',
         'web-bundle',
+        'web-page-js',
+        'handle',
+        'pet-inspector',
+        'pet-store',
+        'lookup',
```

Comment:
There's an outdated comment ~20 lines below this that begins `// The two functions provideValueForFormula and provideValueForFormulaIdentifier`. It should be updated to reflect the removal of `provideValueForFormula`.